scripts/data/embellishTraits.ts

```diff
@@ -50,5 +50,37 @@ function getText(
   const element = $(primarySelector).length
     ? $(primarySelector)
     : $(secondarySelector);
-  return element.parent().next().text().trim().replace(/\\n/g, "");
+
+  return toText(element.parent().next().html());
+}
+
+function toText(html: string | null) {
+  if (html) {
+    while (html.search("<a") >= 0) {
```

Comment:
[blocking] I appreciate the intent here, but the reason for having `cheerio` is so we don't have to try to parse HTML strings ourselves, which is difficult to read and maintain. We should be able to use selectors here if we pass in the `CheerioElement` rather than the HTML string.

Also, if I'm reading this right and am not mistaken, this is using the `while` effectively as an `if` statement, because we don't have any `break`/`continue`/branches without returns, they all hit that last `return` which will break the `while` loop, so only one iteration is ever done - again not great for readability/maintainability.